### PR TITLE
influxdb 2.7.1 - fix ui asset hash

### PR DESCRIPTION
**Pull request by jeffreyssmith2nd**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by jeffreyssmith2nd** [comment]
As a side note, is there a way to make a formula ineligible for `brew bump-formula-pr` to prevent this kind of thing from happening in the future?

**Comment by p-linnane** [comment]
> As a side note, is there a way to make a formula ineligible for `brew bump-formula-pr` to prevent this kind of thing from happening in the future?

No, that's not possible. If the versions of the formula and the ui resource should always be the same then I can add something in the `test` block to ensure that. Let me know.

**Comment by jeffreyssmith2nd** [comment]
> If the versions of the formula and the ui resource should always be the same

Unfortunately that's not something we can guarantee (often times the binary is updated without the UI changing). I'll just keep a closer eye on the PRs next time to see if someone submits one before I do.

**Comment by p-linnane** [comment]
You could always use this action as part of your release process: https://github.com/mislav/bump-homebrew-formula-action

**Comment by jeffreyssmith2nd** [comment]
Ah that looks useful, thanks!

**Review by p-linnane — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

File: Formula/influxdb.rb [hunk 1]
```diff
@@ -44,8 +44,8 @@ class Influxdb < Formula
   # NOTE: The version/URL here is specified in scripts/fetch-ui-assets.sh in influxdb.
   # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
```
Comment (carlocab) [inline Formula/influxdb.rb:45]: Can we read `scripts/fetch-ui-assets.sh` to make sure we have the right version of `ui-assets`?

**Pull request merged**